### DISCUSSION TITLE
NerDL architecture?

### DISCUSSION THREAD

**Discussion by logan-markewich** [JohnSnowLabs/spark-nlp#2906] [General]
Is there any documentation or paper on the architecture of the NerDL model? 

I know it involves a layer of embeddings (BERT, etc.), but what about the rest of the model? Is it just a simple fully connected layer?

**Comment by maziyarpanahi** [comment]
It is explained in this paper: https://arxiv.org/pdf/2011.06315v1.pdf

**Reply by logan-markewich** [reply]
Fantastic, thank you!
